### PR TITLE
Update Batch_Normalization.ipynb

### PR DESCRIPTION
**Pull request by jxtrbtk**
The performance are significantly better on my side when the batch normalization is applied after the ReLU activation function. 
But it seems not to be the case with Sigmoid or Tanh.

### CODE REVIEW

**Comment by abhiojha8** [comment]
This is still debatable. Please see this [StackOverflow post](https://stackoverflow.com/questions/39691902/ordering-of-batch-normalization-and-dropout#:~:text=So%20the%20Batch%20Normalization%20Layer,53%20min%20for%20more%20details.)
